tests/msa_ops/test_msa_ops.py

```
@@ -0,0 +1,1515 @@
+    assert err < 3.5e-2, f"max abs error {err}"
+
+
+def test_sparse_attention_lse():
+    _skip_if_unsupported()
+    from flashinfer.msa_ops import msa_sparse_attention
+
+    q, k, v, idx, cu_q, cu_k = _make_case(
+        2, 4, 2, 16, [80, 50], [1024, 640], torch.bfloat16, seed=30
+    )
+    scale = 1.0 / math.sqrt(128)
+    out, lse = msa_sparse_attention(
```

Ruff (0.15.17)

[warning] Unpacked variable `out` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)